FIAT/enriched.py

```diff
@@ -19,12 +20,12 @@
 from __future__ import absolute_import
 
 import numpy
-from FIAT.finite_element import FiniteElement
+from FIAT.finite_element import CiarletElement
 from FIAT import dual_set
 from copy import copy
 
 
-class EnrichedElement(FiniteElement):
+class EnrichedElement(CiarletElement):
```

Comment:
As far as I understand, this `Enriched` creates a dual by concatenating the nodes of its constituents elements, so the resulting nodes generally won't be a dual basis. Does that still qualify as `CiarletElement`?